### PR TITLE
Customizing hello microverse

### PR DESCRIPTION
**Pull request by Fsher07**

### CODE REVIEW

**Comment by Fsher07** [comment]
I did not understand Stylelint error. I searched it but couldn't find a way to fix it.

File: style.css [hunk 1]
```diff
@@ -0,0 +1,2 @@
+h1 { color: rgb(47, 127, 192); }
+
```
Comment (VICTOR-CHUKWU) [inline style.css]: - [ ] @Fsher07  nice job so far The Stylelint linter is not passing. Kindly fix the problem before asking for another review.

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @Fsher07  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Fsher07** [comment]
I fixed The Stylelint errors.

**Review by elmar8287 — APPROVED** [review]
Hi @Fsher07 , 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**